server/index.js

```diff
@@ -3,6 +3,7 @@ const express = require('express');
 const Path = require('path');
 const bodyParser = require('body-parser');
 
+const app = express();
```

Comment:
This happens further down in the file already. Remove this and it should be good